app/src/main/res/values/strings.xml

```diff
@@ -580,29 +580,34 @@
     <string name="storage_manager_permission_rationale">In Android 11, you need to allow a special storage permission</string>
     <string name="close_app_text">Close App</string>
     <string name="allow_text">Allow</string>
+    <string-array name="faq_categories">
```

Comment:
Placeholder Categories. Once again, this isn't a final product, just for purposes of a school assignment but a good starting point

Reply:
The current repository is a final product and we want actual informative chnages here. We can't add placeholder changes here currently.